src/components/Description.js

```diff
@@ -0,0 +1,50 @@
+import { useDevCycleClient } from '@devcycle/react-client-sdk';
+
+function Description() {
+  const devcycleClient = useDevCycleClient()
+
+  const variation = devcycleClient.config.features['hello-togglebot']?.variationName ?? 'Default'
+
+  return (
+    <div className="App-description">
+      {getMessage(variation)}
+      <a
+        className="App-link"
+        href="https://docs.devcycle.com/sdk/client-side-sdks/react/"
+        target="_blank"
+        rel="noopener noreferrer"
+      >
+        DevCycle React SDK Docs
+      </a>
+    </div>
+  );
+}
+
+const getMessage = (variation) => {
+  if (variation.toLowerCase() === 'default') {
```

Comment:
we only have the 'wink' and 'spin' variations in the current demo feature so i'm removing 'spin' and adding a base variation in [this pr](https://github.com/DevCycleHQ/ttroi/pull/2697) 

i'm thinking it should be called 'Base Variation' instead of 'Default' to avoid confusion with default variable values though - what do you think?

Reply:
I think the spin and wink variations make sense, this "default" is intended for the case that a variation isn't being served from the SDK (so it would be the default values)

Reply:
oh gotcha, that makes sense